VCAC-A/script/setup_access.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash -e
 
-ssh-keygen -q -N ""  < /dev/zero
+ssh-keygen -q -N ""
```

Comment:
Don't understand why removing </dev/zero solves the key already exist issue.

Reply:
if test ! -f ~/.ssh/id_rsa; then
    cat /dev/zero | ssh-keygen -q -N ""
    echo
fi

Reply:
If key exists,  command "ssh-keygen -q -N "" </dev/zero" exits with non-zero error. Since -e presents in the script, the script exits without continuing copying key to VCAC-A.

Adding "if test ! -f ~/.ssh/id_rsa; then" does fix the issue.